openedx/features/edly/tests/test_middleware.py

```diff
@@ -0,0 +1,80 @@
+"""
+Unit tests for Middleware.
+"""
+from testfixtures import LogCapture
+from django.conf import settings
+from django.test import TestCase
+from django.test.client import Client, RequestFactory
+
+from openedx.features.edly import cookies
+from openedx.features.edly.tests.factories import (
+    EdlySubOrganizationFactory,
+    EdlyUserFactory,
+    SiteFactory,
+)
+
+LOGGER_NAME = 'openedx.features.edly.middleware'
+
+
+class EdlyOrganizationAccessMiddlewareTests(TestCase):
+
+    def setUp(self):
+        self.user = EdlyUserFactory()
+        self.request = RequestFactory().get('/')
+        self.request.user = self.user
+        self.request.site = SiteFactory()
+
+        self.client = Client(SERVER_NAME=self.request.site.domain)
+        self.client.login(username=self.user.username, password='test')
+
+    def _create_edly_sub_organization(self):
```

Comment:
remove this method as its just one liner and also its used only once in this test class.

Reply:
Done